wat

```diff
@@ -1,6 +1,10 @@
 #!/bin/sh
 log_lines() {
-  local regex='^.*\(starting full system upgrade\|upgraded.*\)$'
+  local installed=''
+  if [ "x$SHOW_INSTALLED" == "x1" ]; then
```

Comment:
Also, can you pass this value into the function and set it as a new `local` rather than relying on something from the outer scope?